receiver/udp.go

```diff
@@ -95,6 +119,14 @@ func (ud *UDP) Start() error {
 			ud.Threads = 20
 		}
 	}
+	if ud.EmitStats.Duration == 0 {
+		ud.EmitStats.Duration = 10 * time.Second
```

Comment:
Hmm, default emit frequency on a module should probably align with default emit frequency on the receiver (e.g. either 3s og 10s, not one of each)?